src/components/footer.jsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import {
+  Footer as FooterUSWDS,
+  FooterNav,
+  Grid,
+  Logo,
+} from '@trussworks/react-uswds';
+import { useTranslation } from 'react-i18next';
+import { Link } from '.';
+
+import logoImg from '../images/logos/cto_logo_horizontal_white.png';
+
+function Footer() {
+  const { t } = useTranslation();
+  const primaryLinks = ['/link-one', '/link-two', '/link-three', '/link-four'];
```

Comment:
I think it would be best to leave out the changes in the components until my PRs are merged. Once they're merged, you can merge dev into your branch to make sure the changes are made in the components and then create the necessary .jsx files.